docs/reference/transform/checkpoints.asciidoc

```diff
@@ -55,6 +55,47 @@ TIP: If the cluster experiences unsuitable performance degradation due to the
 {transform}, stop the {transform} and refer to <<transform-performance>>.
 
 
+[discrete]
+[[sync-field]]
+== Using the ingest timestamp for syncing the {transform}
+
+In most cases, it is strongly recommended to use the ingest timestamp of the 
+source indices for syncing the {transform}. This is the most optimal way for 
+{transforms} to be able to identify new changes. It can be set by using an 
```

Comment:
What about something like:

If you data source follows the ECS standard (<- link) you might already have a field called `event.ingested`. If you do, you can skip the next section and use `event.ingested` as `sync`.`time`.`field` property of your {transform}. If you don't have a `event.ingested` field or that field isn't populated, here is how to add it.

Reply:
I amended the text based on the suggestion in [f69dc50](https://github.com/elastic/elasticsearch/pull/87650/commits/f69dc506a7c4cba95927b1d1644c8d3b7afda320).